packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -322,6 +323,91 @@ describe('cluster', () => {
     });
   });
 
+  test('cluster with inline parameter group', () => {
+    // GIVEN
+    const stack = testStack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+
+    // WHEN
+    new DatabaseCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA,
+      credentials: {
+        username: 'admin',
+      },
+      parameters: {
+        locks: '100',
```

Comment:
Can we make it something silly, like `'cluster_locks'` and `'instance_locks'`, instead of `100` and `200`, which are a little more tricky to remember which is which in each test?

Reply:
At least now it is a valid value